integration/e2emimir/client.go

```diff
@@ -248,6 +250,136 @@ func (c *Client) QueryRangeRaw(query string, start, end time.Time, step time.Dur
 	return c.DoGetBody(addr)
 }
 
+// RemoteRead runs a remote read request against the response.
```

Comment:
[nit] "against the response" looks a typo. At least I don't understand what you mean here.